libretro.cpp

```diff
@@ -4680,6 +4694,9 @@ void retro_deinit(void)
 
 unsigned retro_get_region(void)
 {
+   // simias: should I override this when fast_pal is set?
+   //
+   // I'm not entirely sure what's that used for.
    return content_is_pal ? RETRO_REGION_PAL : RETRO_REGION_NTSC;
 }
```

Comment:
Can somebody tell me if this is correct? I'm not sure what this is used for, as far as I can tell it's purely a frontend thing.

Reply:
Not sure what this is used for either, maybe it's a carryover from an older version of the libretro API?

Reply:
Asked around, and Autechre says it comes from back when libretro started off as a SNES API. He says he doesn't think it's used right now by RetroArch.

I'm not sure what old frontends are doing with the info being returned by this function.  My guess is something related to timing and aspect ratio, which would probably not work entirely properly with fast PAL hacking. I think it's probably safe to leave as-is with the comments provided until there's a demonstrable problem that occurs with this.